provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -270,21 +275,47 @@ def __init__(
                 opts=pulumi.ResourceOptions(parent=self.function),
             )
             aws.lambda_.Permission(
-                resource_name=f"{resource_name}-cloudwatch-invoke-permission",
+                f"{resource_name}-cloudwatch-invoke-permission",
                 action="lambda:InvokeFunction",
                 function=self.function.arn,
                 principal="events.amazonaws.com",
                 source_arn=rule.arn,
                 opts=pulumi.ResourceOptions(parent=rule),
             )
             aws.cloudwatch.EventTarget(
-                resource_name=f"{resource_name}-keep-warm-target",
+                f"{resource_name}-keep-warm-target",
                 arn=self.function.arn,
                 input=json.dumps({"keep-warm": True}),
```

Comment:
Could you please rename all 3 resources to f"{resource_name}-keep-warm" ? (to be consistent with "-schedule" below)